models/quote.js

```diff
@@ -41,7 +51,9 @@ Object.assign(quote.methods, {
 
   trackPublishing(type) {
     if (type !== 'public') return;
```

Comment:
Magic string:
![](https://media.giphy.com/media/l4FGIgsVPdoRd2wbS/200w.gif)

Use some sort of map or enum :wink: